### PR TITLE
Adds a function to enrich a DF from AmazonGeoTiffs

### PR DESCRIPTION
**Pull request by benwulfe**
Adds a function to enrich a DF from AmazonGeoTiffs

### CODE REVIEW

File: dataset.py
Comment (rothn) [inline dataset.py:48]: Why are we using `getattr(row, "lat")` as opposed to `row["lat"]`?

File: raster.py
Comment (rothn) [inline raster.py:2]: Why is an IPython import in a library?

File: raster.py
Comment (rothn) [inline raster.py:29]: What is this for?

**Review by rothn — CHANGES_REQUESTED** [review]
Requested changes are nits. Feel free to merge if you don't have the capacity to address them.

File: dataset.py
Comment (VimWizardVersace) [inline dataset.py:53]: very nitty, but this will append the new features to the beginning of the dataframe. the docstring kind of implies the order is reversed

File: raster.py
Comment (VimWizardVersace) [inline raster.py:78]: I'm guessing you're checking if gdrive is already mounted? If so, can you add a comment explaining that? It is not immediately obvious

Comment (benwulfe) [reply raster.py:78]: this is unrelated to this PR and only exists because a previous PR hasnt been merged yet

File: dataset.py
Comment (benwulfe) [reply dataset.py:53]: will update comment

File: dataset.py
Comment (benwulfe) [reply dataset.py:48]: fixed

File: dataset.py
Comment (benwulfe) [reply dataset.py:53]: XGB at least wants to keep the labels at the end.  so updated the docstring

File: raster.py
Comment (benwulfe) [reply raster.py:78]: unrelated to this change.  i'm waiting for my other PR to get approval.

File: raster.py
Comment (benwulfe) [reply raster.py:29]: this is the name of the raster, which becomes the name of the column when a feature is added

File: dataset.py
Comment (benwulfe) [reply dataset.py:48]: actually, because its a tuple, you use getattr.  or at least row["x"] doesnt work for me

File: raster.py
Comment (benwulfe) [reply raster.py]: removed.  Not sure how that got added.

**Review by rothn — APPROVED** [review]

**Review by VimWizardVersace — APPROVED** [review]

File: raster.py
Comment (erickzul) [inline raster.py:87]: Did we add a test case for this?

**Review by erickzul — CHANGES_REQUESTED** [review]

File: raster.py
Comment (benwulfe) [reply raster.py:87]: Yes, its in the raster ipynb

**Pull request merged**
